libs/asset-editor/src/lib/components/asset-editor-tab-general/asset-editor-tab-general.component.html

```diff
@@ -163,42 +163,70 @@
     </div>
     <div class="flex flex-column form-column mr-8">
       <div class="flex flex-column bg-white mb-4 py-4 px-6 max-h-full">
-        <ng-container *ngIf="_form.controls['assetFiles'].value.length + _form.controls['newFiles'].value.length !== 0">
+        <ng-container *ngIf="form.controls['assetFiles'].value.length + form.controls['newFiles'].value.length !== 0">
           <div class="font-bold mb-4 mt-2" translate>edit.tabs.general.files</div>
           <div class="max-h-full overflow-y-scroll overflow-x-hidden -mr-5">
             <ul class="mt-0 mb-4 p-0">
               <li
-                *ngFor="let file of _form.controls['assetFiles'].value; let index = index; let last = last"
-                class="list-none bg-grey-00 py-2 pl-4 file-grid no-status mb-2"
-                [ngClass]="{
-                  'py-2': !file.willBeDeleted,
-                  'py-4': file.willBeDeleted,
-                  'no-status': !file.willBeDeleted
-                }"
+                *ngFor="let file of form.controls['assetFiles'].value; let index = index; let last = last"
+                class="list-none bg-grey-00 py-2 pl-4 file-grid mb-2"
               >
-                <div class="ellipsis">{{ file.fileName }}</div>
+                <div class="name ellipsis">{{ file.name }}</div>
 
-                <button
-                  *ngIf="!file.willBeDeleted; else deleteButtonEmpty"
-                  asset-sg-icon-button
-                  class="shrink-0 ml-2"
-                  [disabled]="_disableAll$ | push"
-                  (click)="_deleteFile(file.fileId)"
-                >
-                  <svg-icon key="delete"></svg-icon>
-                </button>
-                <ng-template #deleteButtonEmpty>
-                  <span></span>
+                <ng-container *ngIf="file.willBeDeleted; else fileFormFields">
+                  <div *ngIf="file.willBeDeleted" class="notice text-sm text-grey-08" translate>
+                    edit.tabs.general.willBeDeleted
+                  </div>
+                </ng-container>
+
+                <ng-template #fileFormFields>
+                  <div class="type">
+                    <mat-select [value]="file.type" (valueChange)="updateFileType(file, $event)">
+                      <mat-option class="ellipsis" value="Normal">
```

Comment:
The legal docs are to be shown and edited at a different place than normal files, see issue - so we dont need a select like this because we implicitly always know the type